beacon-chain/sync/subscriber_blob_sidecar.go

```diff
@@ -20,6 +22,13 @@ func (s *Service) blobSubscriber(ctx context.Context, msg proto.Message) error {
 		return err
 	}
 
+	s.cfg.operationNotifier.OperationFeed().Send(&feed.Event{
+		Type: opfeed.BlobSidecarReceived,
+		Data: &opfeed.BlobSidecarReceivedData{
+			Blob: b,
+		},
+	})
+
```

Comment:
Could you move this after `SendNewBlobEvent`? `SendNewBlobEvent` is a critical component of block processing - the data availability check relies on it - so I'd rather keep it right after the db call.